dockers/docker-database/http-server

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+#
+###########################################################################
+# http-server
+# 
+# Python HTTP Server service.
+# This service runs based on the configuration provided in
+# device/<platform>/chassisdb.conf
+#
+# Configuration parameters provided by chassisdb.conf are as follows:
+#
+# start_http_server: if "yes" or "1", http-server will be started on this
+#     node
+# chassis_db_address   : IP address for http-server. This is the same IP used
+#  for the chassis db server
+# http_server_port : Port to bind to, default: 8000
+# http_server_dir  : HTTP server home directory path, default: /var/www/
+#
+###########################################################################
+# Copyright (c) 2021-2022 Cisco Systems, Inc. and its affiliates.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+
+# Example of config:
+'''
+start_http_server=yes
+chassis_db_address=127.0.0.10
```

Comment:
>127.0.0.10 [](http://example.com/codeflow?start=19&length=10)

hard-coded IP?

Reply:
This is an example of the config. The chassis_db_address is used for redis_chassis.server and is hardcoded as of today, provided by chassisd.conf file by vendor in <platform> dir.
This http-serevr use the same IP to bind to service requests on internal midplane network.